src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -268,10 +268,17 @@ def getReplicaInfoForBlocks(self, **kwargs):
             inputDids.append({"scope": kwargs["scope"], "type": "DATASET", "name": block})
 
         resultDict = {}
-        for item in self.cli.list_dataset_replicas_bulk(inputDids):
-            resultDict.setdefault(item['name'], [])
-            if item['state'].upper() == 'AVAILABLE':
-                resultDict[item['name']].append(item['rse'])
+        if kwargs['deep']:
```

Comment:
please reflect the addition in the deep argument in the docstring of the method

Reply:
And you probably need a default value. Something like:
```
kwargs.setdefault("deep", False)
```

Reply:
I'll add the docstring and default value.

Reply:
thanks @germanfgv